### ISSUE TITLE
[IDEA] Community Coin

### ISSUE THREAD

**Issue by adamhannigan** [sunflower-land/sunflower-land#723]
The purpose of this proposal is to explore a community coin incentive for contributors to the project.

**Community Coin**

Being an open source and community driven project, we have a vast amount of contributors of the project who have been involved as a passion project and learning about Web3 development. We currently have artists and designers who have been helping the project for 4+ months. They have built up a lot of contextual information about the project and we are hoping to keep them engaged.

A crypto-based game also relies on other decentralised based contributors such as liquidity providers and partners.

**Purpose of the coin**

The coin will be used as an in-game currency for special items and decorations. Receivers can use these tokens to craft special items or withdraw them to their personal wallets.

## Distribution

**Being entitled to a reward is never a good recipe for building a long lasting community.** We want people excited to be part of this community and the potential of the future iterations. There will be no guarantee when and how these tokens will be distributed. Based on our past record, we have always delivered NFTs + incentives at random for contributors of the project. The MVP of a community coin would work on a similar trust structure.

Following contributions would be the target of the community coin

- Liquidity Providing (Wishing Well)
- Code contribution
- Code reviews
- Designing art
- Music contribution
- Documentation contribution
- Discord moderation
- Community ideas + moderation

![community_coin](https://user-images.githubusercontent.com/11745561/165718684-38fddd8b-29c8-403a-be71-21c096793eda.png)

Artwork by @[Vergelsxtn](https://github.com/Vergelsxtn) in #233

**Comment by SteveChickenCX** [comment]
sounds great, I'm all in for finding more way to incentive the support of our community members :)

**Comment by Netherzapdos** [comment]
Having a lot of fun surfing the github so many new proposals coming in 😂 

I like the idea of giving out the gifts to contributors at random. This pushes the idea that contributors don't contribute in order to get guaranteed free stuff, but rather they get "gifts" from the devs here and there as a thanks to help building the game. This is actually how I explain to people who complain why artists/devs are getting free stuff 😂 They aren't rewarded for their work, rather they receive thanks from the Devs for contributing to the development of the game. So a suggestion of mine would be to phrase it similar to that when/if this feature goes live, as a disclosure of sorts I guess, cause I can already sense some salty people 👀 

Though I do believe some will still try to push the narrative that contributors just contribute in order to get free stuff, not sure if it will be a major issue in the future, but should be taken note of i think.

Kinda similar to my previous thought, what kind of items are the token holders expecting to craft? I'd imagine there might be a small part of the community who is against this idea primarily because contributors are "getting the good stuff" while normal players don't get any. Depends entirely tho on what kind of items will be available for the contributors. But then again, not sure if it will be a major issue.

Lastly, upon reading the statement "or withdraw them to their personal wallets.", will contributors be able to trade/sell these tokens? If you can, then I feel like it shouldn't be withdrawable 🤔 It would defeat the purpose of the token since these are gifted directly to contributors.

Anyways, great stuff!

**Comment by Stevew00dy** [comment]
I think we need to look at 3 perspectives to this

1, The core team 
2, The community
3, The economy

1. The Core Team
From the core team perspective. I understand wanting to reward those who are helping to build the game, this is a noble feat and one that has ensured the engagement we have exists today. I 100% agree with the model to create a way to give back to those who help move the project forward, my concern from this perspective is the additional admin caused as a result. From education, to support and also building, monitoring and securing this additional layer. 

Initially I wonder if it needs to be something new or could it be attached to something already existing. 

2, The community. 
You already touched on the drive people have and how it could become an unhealthy expectation so I'm glad it's being considered, I also wonder if the perception of a coin is the right one. It almost makes the game sound like it has a duel economy which could drive a narrative that people will want to start trading and speculating on it's value. I think it could be considered to have this as an NFT but rather than calling it a coin (although I do love community coin) I wonder if it should be labelled as a Contributor Gift Box - This removes the connection to the coin and perception of it being traded like an asset and more into the thoughts of it being a gift, not expected and for the contributors rather than the community - which is for whom it is designed. I also think that to airdrop these in game would make for a nice UI popup, that would suprise people when they login to find out they can open this box and see what is inside. It allows you to tweak the % based on the contribution and also creates a fun and engaging experience without the feeling of another economy within the game which might confuse some (specially those who dont have it)

3, The economy
Simply put - how to we deliver the consistent value of the asset. Would it be a 1:1 trade on in game items regardless of price, or would it have value that needs to be maintained via supply and demand. Would it be an NFT or an NTT. Do you want contributors to see this to others on the open market. I almost think that a points based system attached to the wallet (incase the user sells the farm) because these points belong to a person and not another. It becomes a status symbol to the work they added to the project and the points can be redeemed for in game assets. I personally think this aspect can be managed from the game interface itself and does not need to live on chain.

**Comment by adamhannigan** [comment]
@Netherzapdos Interesting perspective around locking the items to the farm! I love it. Especially since these items would be a "flex" of your contributions to the project, it does not really make sense that you would sell it to someone else who did not contribute.

@Stevew00dy Great points. Starting to think that a tradable coin does not make too much sense here. I love the idea of a nice treasure chest popping up for our contributors at random! ❤️

**Comment by StockSavage** [comment]
> @Netherzapdos Interesting perspective around locking the items to the farm! I love it. Especially since these items would be a "flex" of your contributions to the project, it does not really make sense that you would sell it to someone else who did not contribute.
> 
> @Stevew00dy Great points. Starting to think that a tradable coin does not make too much sense here. I love the idea of a nice treasure chest popping up for our contributors at random! ❤️ I also believe that showing some items in the shop as requiring a "community coin" is also powerful as other players will understand where these items come from 🤗

I agree with this completely. I'd love to have something like this that actually rewards community members and dedicated players. Locking it to the farm is such a good way to go about it. Something you can't buy, but earn through your support for the project. I love it.

**Comment by adamhannigan** [comment]
@DonatasDee Awesome feedback. Agreed it would be hard to police and would potentially cause people abusing the system.

Fair feedback around the proposals. Most of my Github issues are created just to move unorganised Discord conversations into a organised area.

I have changed the title to [IDEA] instead to distinct it from an actual [PROPOSAL]. What do you think?

**Comment by DonatasDee** [comment]
> @DonatasDee Awesome feedback. Agreed it would be hard to police and would potentially cause people abusing the system.
> 
> Fair feedback around the proposals. Most of my Github issues are created just to move unorganised Discord conversations into a organised area.
> 
> I have changed the title to [IDEA] instead to distinct it from an actual [PROPOSAL]. What do you think?

Definitely makes it more clear, because when the developer throws out a proposal it used to feel pretty concrete optically, even if there was plenty lee-way to change your mind or offer other solutions.
